src/components/Dashboard/TodoList.vue

```diff
@@ -514,35 +568,39 @@ export default {
       return true
     },
 
-    isNew (item) {
-      return item.status === FilingStatus.NEW
+    isNew (task) {
+      return task.status === FilingStatus.NEW
     },
 
-    isDraft (item) {
-      return item.status === FilingStatus.DRAFT
+    isDraft (task) {
+      return task.status === FilingStatus.DRAFT
     },
 
-    isPending (item) {
-      return item.status === FilingStatus.PENDING
+    isPending (task) {
+      return task.status === FilingStatus.PENDING
     },
 
-    isError (item) {
-      return item.status === FilingStatus.ERROR
+    isError (task) {
+      return task.status === FilingStatus.ERROR
     },
 
-    isPaid (item) {
-      return item.status === FilingStatus.PAID
+    isPaid (task) {
+      return task.status === FilingStatus.PAID
     },
 
-    isCompleted (item) {
-      return item.status === FilingStatus.COMPLETED
+    isCompleted (task) {
+      return task.status === FilingStatus.COMPLETED
     },
 
-    confirmDeleteDraft (item) {
+    isCorrection (task) {
+      return task.type === FilingTypes.CORRECTION
+    },
+
+    confirmDeleteDraft (task) {
       // open confirmation dialog and wait for response
       this.$refs.confirm.open(
         'Delete Draft?',
-        'Delete your ' + item.draftTitle + '? Any changes you\'ve made will be lost.',
+        'Delete your ' + task.draftTitle + '? Any changes you\'ve made will be lost.',
```

Comment:
Oops, you do need `draftTitle` after all.

Also on line 664 below.

Reply:
Corrected!